components/Member/Profile/ContactInfo.js

```diff
@@ -0,0 +1,280 @@
+      }
+    }
+  }
+`;
+
+const UPDATE_MEMBER = gql`
+  mutation updateMember($profile: ProfileUpdateInput!) {
+    members {
+      member {
+        update(profile: $profile) {
+          id
+          profileSlug
+          firstName
+          lastName
+          email
+          canFeature
+          createdAt
+          lastUpdatedAt
+        }
+      }
+    }
+  }
+`;
+
+const FlexFormRow = styled(FormRow)`
+  display: flex;
+  justify-content: space-between;
+
+  label {
+    flex-grow: 2;
+    &:not(:last-child) {
+      padding-right: 2rem;
+    }
+  }
+`;
+
+const ContactInfoForm = () => {
+  const { loading, error, data } = useQuery(GET_MEMBER);
+  const [updateMember] = useMutation(UPDATE_MEMBER, {
+    onCompleted: () => {
+      Router.push('/member/online-presence');
+    },
+    onError: updateError => {
+      console.log('Error updating member', updateError);
+    },
+  });
+
+  if (loading) return 'Loading...';
+  if (error) return `Error! ${error.message}`;
+
+  const {
+    city,
+    company,
+    country,
+    email,
+    firstName,
+    isOver18,
+    jobTitle,
+    lastName,
+    mobilePhone,
+    // profileImage,
+    profileSlug,
+    state,
+  } = data.members.me;
+
+  return (
+    <Formik
+      initialValues={{
+        city,
+        company,
+        country,
+        email,
+        firstName,
+        jobTitle,
+        lastName,
+        mobilePhone,
+        state,
+        // image: null,
+      }}
+      validationSchema={Yup.object({
+        firstName: Yup.string()
+          .min(3, 'Must be at least 3 characters')
+          .required('Required'),
+        lastName: Yup.string()
+          .min(3, 'Must be at least 3 characters')
+          .required('Required'),
+        email: Yup.string()
+          .email('Invalid email address')
+          .required('Required'),
+        mobilePhone: Yup.string()
+          .matches(RegularExpressions.phoneRegExp, 'Phone number is not valid')
+          .nullable(),
+        // showEmailAddressOnProfile: Yup.bool(),
+        // showMobilePhoneOnProfile: Yup.bool(),
+        city: Yup.string().nullable(),
+        state: Yup.string().nullable(),
+        country: Yup.string().nullable(),
+        company: Yup.string().nullable(),
+        jobTitle: Yup.string().nullable(),
+        // image: Yup.mixed()
+        //   .test(
+        //     'fileSize',
+        //     'File Size is too large',
+        //     value => !value || value.size <= 300000,
+        //   )
+        //   .test(
+        //     'fileType',
+        //     'Unsupported File Format',
+        //     value =>
+        //       !value ||
+        //       ['image/jpg', 'image/jpeg', 'image/gif', 'image/png'].includes(
+        //         value.type,
+        //       ),
+        //   ),
+      })}
+      onSubmit={(values, { setSubmitting }) => {
+        setTimeout(() => {
+          console.log('values', values);
+          const profile = { ...values, isOver18, isDeactivated: false };
```

Comment:
@csell5 can't run an update on a member without passing `isOver18` and is `isDeactivated`

Reply:
that was *just* fixed. they're gone.